sum.js

```diff
@@ -0,0 +1 @@
+const sum = (a, b) => a + b;
```

Comment:
Che, por qué no documentaste qué hace esta función.
```suggestion
/*
* Recibe dos numeros a y b y retorna la suma
*/
const sum = (a, b) => a + b;
```